animareflection/ui
# Drawer Component

## Description

##### Task link: https://trello.com/c/iBERH60r/108-drawer-component

Added custom `Drawer` component with `trigger`, `placement`, `title`, and `description` props. Also added primitive exports to allow for other users to build and customize `Drawer` components outside of AR branding/usage.

NOTE: Also created custom `Icon` component in this PR to allow for ease of use with `react-icons` in storybook and complete components.

## Test Steps

1) Verify that everything renders appropriately in storybook
2) Verify that all tests pass
3) Verify that `Drawer` usage in downstream app works as expected
4) Verify that `primitives` module gets bundled appropriately, and that users may construct custom `Drawer` components in a downstream app.
